src/ORM/Table.php

```diff
@@ -2587,6 +2587,7 @@ public function callFinder(string $type, SelectQuery $query, array $options = []
     {
         assert(empty($options) || !array_is_list($options), 'Finder options should be an associative array not a list');
 
+        /** @var array<string, mixed> $options */
```

Comment:
There is a psalm but that cause the assert to change the type to possibly be the one we're asserting it isn't.